docs/platform/what-is-tizen/profiles/iot.md

```diff
@@ -0,0 +1,100 @@
+# Tizen IoT
+
+Tizen IoT profile allows you to make various kinds of IoT devices and develop applications for the IoT devices.
+Tizen IoT Headed and Headless binaries support any type of IoT devices based on the Linux kernel.
+
+![iot_building_block](media/iot_building_block.png)
+
+
+## ABB: API building block
+
+Tizen IoT provides a building block pool of components mainly based on the [Tizen native API sets](../../../application/native/api/wearable/5.5/index.html).
+
+Custom IoT images can be created by combining the building blocks in various ways as per the requirements.
+
+Required packages are installed based on the dependencies related to the specified building blocks.
+
+
+## Types of building block presets
+
+### IoT core
+- The minimum set of bootable modules that make up the Tizen platform image for IoT devices.
+- The preset contains core components that are required for a basic system image to boot.
+
+### Headless
+- The set of modules that make up the Tizen platform image for headless IoT devices without a display.
+- The preset contains IoT core presets with other components specifically required for headless image.
+- Example of specific components for headless image includes Audio moduldes, Bluetooth modules, and so on.
+
+### Headed
+- The set of modules that make up the Tizen platform image for headed IoT devices with a display.
+- The preset contains IoT core presets with other components specifically required for headed image.
+- Example of specific components for headed image includes Notification API, graphic modules, and so on.
+
+## MIC image creator
+
+MIC is an image creator that is used to create images for Tizen. 
+
+The tool offers three major functions as follows:
+- create an image in a specific format
+- chroot into an image
+- convert an image to another format
+
+For more information on the tool and its functions, see [MIC Image Creator](https://docs.tizen.org/platform/reference/mic/mic-overview/).
+
+In this document, we will focus on creating the image using MIC.
+
+## Create image using MIC
+
+Before you create an image using MIC, ensure that the following are supported:
+- OS: Ubuntu 14.04 LTS or higher
+- Git Build System(GBS) and Image Creator(MIC) for Tizen developers
+
+    ```shell
+    $ echo "deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_16.04/ /" | sudo tee /etc/apt/sources.list.d/tizen.list
+    $ sudo apt-get update
+    $ sudo apt-get install gbs mic
+    ```
+
+To create a custom Tizen IoT image, follow these steps:
+
+1. Download a kickstart(.ks) file from the [Tizen download site](http://download.tizen.org/snapshots/tizen/unified/). 
+    - You may choose the file for the binary that is most similar to the custom image you are trying to create.
+
+2. Edit the kickstart file in accordance with the requirements of the custom image. 
+    - You can add the name of the required building blocks under the %packages section of the kickstart file.
```

Comment:
Please add a colon instead of a period as it is followed by a code.

- You can add the name of the required building blocks under the %packages section of the kickstart file:

Reply:
done :)